com/metacube/shoppingcart/enums/StatusEnum.java

```diff
@@ -0,0 +1,5 @@
+package com.metacube.shoppingcart.enums;
+
+public enum StatusEnum {
```

Comment:
You are not using these enums anywhere in this assignments, so either you should use these or delete them.